waspc/test/AnalyzerTest.hs

```diff
@@ -82,7 +84,17 @@ spec_Analyzer = do
                 "job BackgroundJob {",
                 "  executor: PgBoss,",
                 "  perform: {",
-                "    fn: import { backgroundJob } from \"@ext/jobs/baz.js\"",
+                "    fn: import { backgroundJob } from \"@ext/jobs/baz.js\",",
+                "    executorOptions: {",
```

Comment:
When I previously mentioned `executorOptions` as a name, I thought that would be used ad top level, next to `executor`, and that these two `options` that are level deeper would be just `options`. You named them `executorOptions` -> how do you feel about that, better than `options`? I think I am missing context to know myself what is more suitable so just wanted to check with you.

Reply:
I kinda stole the idea from the other context because I really liked being explicit here. :D Now that we are explicitly listing the executor under those options, I thought it was fitting to be a bit more explicit and self-document even more that these are, in fact, executor-specific options we pass along. I could also imagine at some point we have Wasp-specific options we may want to call `options` in the future as well. So I figured the extra verbosity for clarity/self-documentation was appropriate.

Reply:
Sounds good!